src/SharedPipelineComponents/IlrSubmissionEvents/DeploySubmission/copy mappings/DasSubmissionEventsCopyToDedsMapping.xml

```diff
@@ -105,6 +105,14 @@
           <Source>EPAOrgId</Source>
           <Destination>EPAOrgId</Destination>
         </ColumnMapping>
+        <ColumnMapping>
+          <Source>GivenNames</Source>
+          <Destination>GivenNames</Destination>
+        </ColumnMapping>
+        <ColumnMapping>
+          <Source>FirstName</Source>
+          <Destination>FirstName</Destination>
+        </ColumnMapping>
```

Comment:
GivenNames and FirstName? Should one of these be surname?

Reply:
Or FamilyName

Reply:
Yes, of course, sorry.  Pushed a commit to change below.